composer.json

```diff
@@ -0,0 +1,96 @@
+{
+    "name": "prestashop/cli-upgrade-assistant",
```

Comment:
We have 2 composer.json? One in CliUpgradeBundle and this one?

Reply:
Yep, this is done because in the future why not to put this bundle away from this repository! So I need another composer json for this bundle (and create properly the composer autoloader).